### PR TITLE
Implement more tests for event create feature

### PR DESCRIPTION
**Pull request by jordankanghm**
Just some extra test cases for ecreate, can just merge

### CODE REVIEW

**Review by zacwong2151 — APPROVED** [review]

File: src/main/java/seedu/address/model/event/UniqueEventList.java [hunk 1]
```diff
@@ -28,6 +28,14 @@ public class UniqueEventList implements Iterable<Event> {
     private final ObservableList<Event> internalUnmodifiableList =
             FXCollections.unmodifiableObservableList(internalList);
 
+    /**
+     * Returns the event in the event storage with the {@code eventName}.
+     */
+    public Event get(EventName eventName) {
```
Comment (tiongMax) [inline src/main/java/seedu/address/model/event/UniqueEventList.java:34]: Maybe can be more specific on the name, i.e. getEvent

File: src/test/java/seedu/address/logic/commands/VolunteerEditCommandTest.java [hunk 2]
```diff
@@ -55,28 +55,29 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
...
+    //        String expectedMessage = String.format(VolunteerEditCommand.MESSAGE_EDIT_VOLUNTEER_SUCCESS,
+    //                                                                        Messages.format(editedVolunteer));
+    //
+    //        Model expectedModel = new ModelManager(new EventStorage(model.getEventStorage()),
+    //                                                new VolunteerStorage(model.getVolunteerStorage()),
+    //                                                new UserPrefs());
+    //        expectedModel.setVolunteer(lastVolunteer, editedVolunteer);
+    //
+    //        assertCommandSuccess(volunteerEditCommand, model, expectedMessage, expectedModel);
+    //    }
```
Comment (tiongMax) [inline src/test/java/seedu/address/logic/commands/VolunteerEditCommandTest.java:80]: Can just delete if these are not going to be used anymore

**Review by tiongMax — COMMENTED** [review]
Can consider my suggestion if deem appropriate

**Review by zacwong2151 — APPROVED** [review]

File: src/main/java/seedu/address/model/event/UniqueEventList.java (hunk 1 above)
Comment (jordankanghm) [reply src/main/java/seedu/address/model/event/UniqueEventList.java:34]: i agree!

**Review by tiongMax — APPROVED** [review]
Lgtm!

**Pull request merged**